atd-vze/src/_nav.js

```diff
@@ -33,6 +33,18 @@ export const navigation = roles => {
       {
         divider: true,
       },
+      {
+        name: 'Reports',
+        url: '#',
+        icon: 'icon-chart',
+        children: [
+          {
+            name: "Inconsistent Crashes",
```

Comment:
Maybe we call this "Inconsistent KSI Counts"? Because its not necessarily the Crash thats inconsistent, its all the related records, specifically the KSI counts across related records.